pkg/test/integration/common/helpers/rbac.go

```diff
@@ -0,0 +1,198 @@
+package helpers
+
+import (
+	rbacapi "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (c *Cluster) ControlClusterRolesAndRoleBindingSetup(namespace string) {
```

Comment:
These cluster Roles and cluster Rolebindings are evolving in nature and change on every PR merge. I think makes sense to rather [deploy the respective files from here](https://github.com/gardener/machine-controller-manager/tree/master/kubernetes/deployment/out-of-tree) instead? Or fetch them from these files?WDYT?

Feel free to make minor changes to those files to suit your test scenario.

Reply:
I have replaced this code to use the yaml files. Thanks for the feedback